examples/calculations/submit_singlepoint.py

```diff
@@ -36,12 +36,13 @@ def singlepoint(params: dict) -> None:
     inputs = {
         "metadata": {"options": {"resources": {"num_machines": 1}}},
         "code": params["code"],
-        "architecture": Str(params["architecture"]),
+        "arch": Str(params["arch"]),
         "structure": structure,
-        "model": model,
         "precision": Str(params["precision"]),
         "device": Str(params["device"]),
     }
+    if model is not None:
```

Comment:
Do we not need a model by this point? (If there's a default, would it not be defined in `load_model`?)

(This applies in the other examples too)

Reply:
there is a default in the calcjob code itself, so the load_model function I made it to return None if None was given as an argument.
But if I input model=None, aiida explodes so I did it this way. 
Probably it owuld be better to have a default in the load_model, the problem is that the load_model wants a url or a path, and if I default a path, then another person using it might not have that path.
Idk if it makes sense

Reply:
Whether we have a default or not is probably a separate question (and I don't have a strong opinion).

My question for now is, since we only set `inputs["model"]` if `model is not None`, what does happen if `model is None` here? In particular, does it give a useful error/warning or still explode in some form?

Reply:
if you set it as None the calcjob accept it as an input and when later it has to do some stuff with it it explode cause it will look for some properties of the ModelData that `None` does not have

Reply:
Sorry, I may be misunderstanding, but with your current change, the model is no longer in the `inputs`, so would something different not happen? Or is `model` itself still passed even if it's not in the `inputs` dictionary?

I don't think it makes sense to let it get far enough to explode, but even more than before, I think this should be addressed separately.

Reply:
If I don't give a model, it will use a default model that is defined inside the calcjob. If I give None, it will take None as model and explode. I have no idea if this makes sense (I would think None and giving nothing would act the same) but that's what it does